frontend/src/pages/CourseSelector/courseDescription/CourseDescription.jsx

```diff
@@ -135,7 +135,7 @@ export default function CourseDescription({ structure }) {
         isUnlocked: true,
         warnings: [],
         handbook_note: "",
-        isAccurate: true,
```

Comment:
Hmm - neither of these is correct tbh. The user can still put the order incorrectly (e.g. the user has done 1511, but only after 1521. This should return false, but instead this will be true - because the order is not correctly considered using this method.)

Reply:
hmm you should look again at the files changed, pretty sure `isAccurate = course.is_accurate` now. Not sure why it's still displayed as `isAccurate: true` here.

Reply:
Yup fair, nvm I misunderstood